tests/test_metadata/test_filemaker.py

```diff
@@ -0,0 +1,46 @@
+from unittest import TestCase
+from src.ftva_etl.metadata.filemaker import (
+    is_series_production_type,
+)
+from fmrest.record import Record
+
+
+class TestFilemaker(TestCase):
+    def test_is_series_production_type(self):
+        # Cherry-picked data from Filemaker PD that should be identified as series
+        valid_test_cases = [
```

Comment:
As found in the previous comment about trimming FM data after splitting, add a (valid) test case here where one of the terms of interest has a leading and/or trailing space, like `"NEWS \rSHORT"`.